frontend-react/src/components/Table/TestTable.tsx

```diff
@@ -24,26 +26,27 @@ const dummyRowTwo = {
  * Table component. Any  */
 export const TestTable = () => {
     const filterManager = useFilterManager();
-    const cursorManager = useCursorManager("firstCursor");
+    const { cursors, hasNext, hasPrev, update } =
```

Comment:
I like how this improves readability. Nice code choice.

Opinion: "update" is a pretty generic name. "updateCursor" might be better? I'm just thinking of how it appears in the code using it.

Reply:
Oops, that needs a label, but I did a similar thing with the filter hooks; you can read them in either way:

```
const cursorManager = useCursorManager()
cursorManager.update(...)

// or

const { ..., update: updateCursor } = useCursorManager()
updateCursor(...)
```

And I like this, because if we do choose not to destructure our response, it's a clean thing to read, like `pages.update()` rather than `pages.updatePages()`, but destructuring means we can always rename it and use without that dot notation, so we'd want to include the property updated in the name, like `updatePages()` instead of that `update()` call being ambiguous.

I'll go in and fix that one, but peek inside `useFilterManager` to see that destructuring and renaming process! :)

Reply:
I would have been fine with leaving it as it was but I like this solution!